src/magneticod/persistence.go

```diff
@@ -226,3 +227,30 @@ func setupSqliteDatabase(database *sql.DB) error {
 
 	return nil
 }
+
+func setupMySQLDatabase(database *sql.DB) error {
+	_, err := database.Exec(
+		`CREATE TABLE IF NOT EXISTS torrents ("
+			id		INTEGER PRIMARY KEY AUTO_INCREMENT,
+			info_hash	BINARY(20) NOT NULL UNIQUE,
+			name		VARCHAR(255) NOT NULL,
```

Comment:
Sampling two instances shows the following:

```sqlite
sqlite> SELECT length(name) FROM torrents ORDER BY length(name) DESC LIMIT 1;
246
sqlite> SELECT count(*) FROM torrents;
2241382
```

```sqlite
sqlite> SELECT count(*) FROM torrents;
2173015
sqlite> SELECT length(name) FROM torrents ORDER BY length(name) DESC LIMIT 1;
330
```

So we should probably set it to something reasonable larger, like 512/1024 I guess.